_data/projects/Firecracker.yml

```
@@ -0,0 +1,10 @@
+name: Firecracker
+desc: CLI that maps business logic, anomaly decider, attacker and visualizer
+site: https://github.com/blst-security/firecracker
+tags:
+  - API-securrity
+  - Rust
+  - CLI
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/username/project/labels/up%20for%20grabs
```

At the moment there are no open issues here, so I'm not sure which label you plan to use:

```suggestion
  link: https://github.com/blst-security/firecracker/labels/help%20wanted
```